Marlin/Configuration.h

```diff
@@ -362,9 +390,9 @@ const bool Z_MAX_ENDSTOP_INVERTING = true; // set to true to invert the logic of
 #define max_software_endstops true  // If true, axis won't move to coordinates greater than the defined lengths below.
 
 // Travel limits after homing (units are in mm)
-#define X_MAX_POS 205
```

Comment:
They were, but I changed them back to their original settings, as they have nothing to do with the pneumatics.